src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -176,7 +292,11 @@ def run(self, *args, **kwargs):
 
 class pt12(Macro):
     """Macro with list of motors followed by list of numbers. Two repeat
-    parameters may defined."""
+    parameters may defined.
+    Usages, ex.:
+    pt12 [[1][3][4]] [[mot1][mot2]]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.